lib/active_remote/attributes.rb

```diff
@@ -1,5 +1,33 @@
 module ActiveRemote
   module Attributes
+    extend ActiveSupport::Concern
+    include ActiveModel::AttributeMethods
```

Comment:
Does this work? I can see specs are passing, but I'd expect this to have to be included in the base class.